.eslintrc.yaml

```diff
@@ -1,2 +1,11 @@
 ---
 extends: "@commercetools-frontend/eslint-config-mc-app"
+settings:
+  import/ignore:
+    # included by default, but replaced if explicitly configured
+    - node_modules
+    # We use `export * from 'react-testing-library';` in `test-utils`, so that
+    # tests can do `import { render, fireEvent }` from '../test-utils'`.
+    # Because we export the whole namespace, the `import/ignore` rule can't find
+    # the named exports. So we disable the rule for imports of that file.
+    - test-utils.js
```

Comment:
I force-pushed these changes. We now have a `test-utils` which exports a configured `render`. This is not necessary for the tests shown in this PR, but it is necessary for later tests which use `IntlProvider`. We can extend the basic `render` fn as needed. For now, I found it important to get the setup right from the beginning, so that all future tests use the `render` fn from `test-utils`.